pokemongo_bot/__init__.py

```diff
@@ -110,17 +110,20 @@ def work_on_cell(self, cell, position, include_fort_on_path):
                 forts = [fort
                          for fort in cell['forts']
                          if 'latitude' in fort and 'type' in fort]
-
-                # Sort all by distance from current pos- eventually this should
-                # build graph & A* it
-                forts.sort(key=lambda x: distance(self.position[
-                           0], self.position[1], x['latitude'], x['longitude']))
-                for fort in cell['forts']:
+           
+                while len(forts) > 0:
+                    # Sort all by distance from current pos- eventually this should
+                    # build graph & A* it
+                    forts.sort(key=lambda x: distance(self.position[
+                               0], self.position[1], x['latitude'], x['longitude']))
+                    fort = forts[0]
                     worker = SeenFortWorker(fort, self)
                     hack_chain = worker.work()
```

Comment:
Shouldn't you set hack_chain to a value after you remove a fort? Like moving a like 121 to line 124